Orange/widgets/visualize/owscatterplot.py

```diff
@@ -619,5 +620,38 @@ def test_main(argv=None):
 
     return rval
 
+
+class Test(unittest.TestCase):
```

Comment:
Gui tests should extend the GuiTest class, which makes sure that all gui tests are run using the same QApplication.

https://github.com/biolab/orange3/blob/master/Orange/widgets/tests/base.py#L8